report/styled_par.go

```diff
@@ -0,0 +1,69 @@
+/*
+ * This example showcases the creation of styled paragraph
+ * The output is saved as styled_paragraph.pdf which illustrates some of the features
+ * of the creator.
+ */
+/*
+ * NOTE: This example depends on github.com/unidoc/unipdf/v3/creator, MIT licensed,
+ *       and github.com/unidoc/unipdf/v3/model,
+ *       Apache-2 licensed.
+ */
+
+package main
+
+import (
+	"log"
+
+	"github.com/unidoc/unipdf/v3/creator"
+	"github.com/unidoc/unipdf/v3/model"
+)
+
+func main() {
+
```

Comment:
Remove extra newline.  Keep style consistent with https://github.com/unidoc/unipdf/wiki/UniPDF-Developer-Guide
also run golint and goimports on the code.

Reply:
Removed